### PR TITLE
added cpp solution to lc:923

### PR DESCRIPTION
**Pull request by Dhoni77**
✅ added cpp solution
✅ update readme

### CODE REVIEW

File: solution/0900-0999/0923.3Sum With Multiplicity/Solution.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Class Solution{
```
Comment (poyea) [inline solution/0900-0999/0923.3Sum With Multiplicity/Solution.cpp:1]:
```suggestion
class Solution{
```

File: solution/0900-0999/0923.3Sum With Multiplicity/README_EN.md [hunk 2]
```diff
@@ -58,10 +58,29 @@ and two 2s from [2,2,2,2] in 6 ways.
...
 ```
 
-### **...**
-
-```
-
+### **C++**
+
+```cpp
+Class Solution{
```
Comment (poyea) [inline solution/0900-0999/0923.3Sum With Multiplicity/README_EN.md:64]:
```suggestion
class Solution{
```

File: solution/0900-0999/0923.3Sum With Multiplicity/README.md [hunk 3]
```diff
@@ -68,10 +68,29 @@ arr[i] = 1, arr[j] = arr[k] = 2 出现 12 次：
...
 ```
 
-### **...**
-
-```
-
+### **C++**
+
+```cpp
+Class Solution{
```
Comment (poyea) [inline solution/0900-0999/0923.3Sum With Multiplicity/README.md:74]:
```suggestion
class Solution{
```

**Comment by Dhoni77** [comment]
done the changes @poyea 👍

**Review by yanglbme — APPROVED** [review]

**Pull request merged**